crates/vault-registry/src/lib.rs

```diff
@@ -754,6 +802,8 @@ pub mod pallet {
                 LiquidationCollateralThreshold::<T>::insert(currency_pair, threshold);
             }
             StorageVersion::<T>::put(Version::V3);
+            let no_release: Option<ClientRelease<T::Hash>> = None;
+            PendingClientRelease::<T>::put(no_release);
```

Comment:
Assuming `PendingClientRelease` is `None` by default I think you can remove these lines.